### ISSUE TITLE
Formatting: Alignment -> Argument List does not work

### ISSUE THREAD

**Issue by winniehell** [deepsky/SQL-Code-Assistant-#13]
IntelliJ 13.1.2
Plugin version: 1.5.13.135

Formatted output:

``` sql
    FUNCTION add_child (
        pParentID GlobalTree.parentID%TYPE,
        pChildVersID GlobalTree.childVersID%TYPE
    ) RETURN GlobalTree.childID%TYPE;
```

Expected:

``` sql
    FUNCTION add_child(    
        pParentID     GlobalTree.parentID%TYPE,
        pChildVersID  GlobalTree.childVersID%TYPE
    ) RETURN          GlobalTree.childID%TYPE;
```

**Comment by deepsky** [comment]
Hi Winnie! Going to have it fixed in the next release. I will appreciate if you report other issue with formatting. 
Regards,
Serhiy

**Comment by deepsky** [comment]
Fixed in 1.6.1 release
